AirCasting/Utils/Constants.swift

```diff
@@ -29,6 +29,8 @@ struct Constants {
         formatter.dateFormat = "HH:mm"
         return formatter
     }()
+    
+    static var isSatelliteMapOn: Bool { UserDefaults.standard.bool(forKey: UserDefaultsKeys.satelliteMapKey) }
```

Comment:
I'm hesitant about putting this into constant since it's not a constant but rather something a user can change.